tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27,pep8
+envlist = py27,py37,pep8
```

Comment:
Shoudn't we take out python 2?

Reply:
The CI still run py27 step in pipeline, I think is better remove completely in another task.

Reply:
Done!